Makefile

```diff
@@ -31,13 +31,22 @@ install-provisioner:
 init-terraform:
 	terraform init -upgrade=true
 
+SSH_CONFIG_DIR := ~/.ssh/config.d
+SSH_CONFIG_FILE := infra-nimbus
+SSH_USERNAME := $$(whoami)
+
+ssh-config:
+	SSH_CONFIG_DIR=$(SSH_CONFIG_DIR) \
+	SSH_CONFIG_FILE=$(SSH_CONFIG_FILE) \
+	SSH_USERNAME=$(SSH_USERNAME) \
```

Comment:
You know, you can just add `export` to the definitions above, and they will be available to the script.

Reply:
I'd recommend making it like this:
```Makefile
ssh-config: export SSH_CONFIG_DIR := ~/.ssh/config.d
ssh-config: export SSH_CONFIG_FILE := infra-nimbus
ssh-config: export SSH_USERNAME := $$(whoami)
	bash scripts/create-ssh-config.sh
```

Reply:
And you don't need to call `bash` if the script already has a shebang.

Reply:
Also please put it at the bottom of the file.